tests/unit/expire.tcl

```diff
@@ -321,7 +321,7 @@ start_server {tags {"expire"}} {
             r set foo1 bar ex 100
             r set foo2 bar px 100000
             r set foo3 bar exat [expr [clock seconds]+100]
-            r set foo4 bar pxat [expr [clock milliseconds]+100000]
+            r set foo4 bar PXAT [expr [clock milliseconds]+100000]
```

Comment:
The first uses PXAT, so get PXAT

Reply:
hi @enjoy-binbin, could I ask why the two positions change?